src/NLog/LogFactory.cs

```diff
@@ -1248,10 +1255,18 @@ private static Logger CreateDefaultLogger(ref LoggerCacheKey cacheKey)
             return newLogger;
         }
 
-        private void LoadLoggingConfiguration(string configFile)
+
+        internal LoggingConfiguration LoadConfiguration(string configFile)
```

Comment:
I think this should be `public`, as the LogFactory is "in sync" with the LogManager. What do you think?

Reply:
Made public as suggested.